packages/api/package.json

```diff
@@ -68,6 +68,8 @@
     "node-forge": "^0.7.1",
     "p-limit": "^1.1.0",
     "querystring": "^0.2.0",
+    "request": "^2.85.0",
```

Comment:
It could be possible to use [got](https://npmjs.com/got), which is already a dependency, instead of request/request-promise, to cut down on dependencies.